lib/rspec/core/configuration.rb

```diff
@@ -484,6 +484,11 @@ def reset
         @formatter_loader = nil
       end
 
+      def reset_reporter
```

Comment:
And I think we should be intentional about it: if we add docs, we are declaring it part of our SemVer public API and committing to support it.  I don't see this needing to be part of our SemVer public API so would recommend `# private`.